### PR TITLE
[ADAM-1889] Run R/Python tests.

### LINKED ISSUE #1889: Jenkins does not run python or R tests
```
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.11: Python APIs 0.24.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
...
[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ adam-python-spark2_2.11 ---
[INFO] Tests are skipped.
```

### PR DESCRIPTION
**Pull request by fnothaft**
Resolves #1889.

Let's see if this does it...

### CODE REVIEW

File: scripts/jenkins-test [hunk 1]
```diff
@@ -213,10 +213,11 @@ then
 	    export SPARKR_SUBMIT_ARGS="--jars ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} --driver-class-path ${ASSEMBLY_DIR}/${ASSEMBLY_JAR} sparkr-shell"
 	    
             # we can run the python build, now that we have a spark executable
+	    # -DskipTests appears to skip the python/r tests?
 	    mvn -U \
 		-P python,r \
-		package \
-		-DskipTests \
+		test \
```
Comment (heuermh) [inline scripts/jenkins-test]: I seem to recall skipping python tests was intentional?

Comment (akmorrow13) [reply scripts/jenkins-test]: Why would we want to skip python tests? This would result in merging broken code.

Comment (heuermh) [reply scripts/jenkins-test]: I dunno, git blame points to commit https://github.com/bigdatagenomics/adam/commit/7e92b9dcd475deecf6ed7c71761dbfcc6b16a9cf and before that commit https://github.com/bigdatagenomics/adam/commit/640c44bae1841217f981899c3a2b793c7ae06dcd and before that commit https://github.com/bigdatagenomics/adam/commit/c34a44004f80d4958df40eed6ae5d3760b8317ae, so it's always been that way

**Comment by akmorrow13** [comment]
@fnothaft do you have residual code for python ReferenceRegion that wasnt merged or should we write this from scratch?

File: scripts/jenkins-test (hunk 1 above)
Comment (fnothaft) [reply scripts/jenkins-test]: Nope, not intentional, just a goof on my end.

**Comment by fnothaft** [comment]
> do you have residual code for python ReferenceRegion that wasn't merged or should we write this from scratch?

Yeah, let me find it. I think it is on my Berkeley laptop.

**Comment by akmorrow13** [comment]
Closing in favor of https://github.com/bigdatagenomics/adam/pull/1892

**Pull request closed without merging**